### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by roydahan**
quick fixes for master and branch-3.1

### CODE REVIEW

**Comment by roydahan** [comment]
@amoskong @aleksbykov @juliayakovlev whoever can review it first...
I want to merge it ASAP though it's not tested...

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2211,13 +2211,13 @@ def _gen_cqlsh_cmd(self, command, keyspace, timeout, host, port, connect_timeout
             auth_params=auth_params, use_keyspace=use_keyspace, timeout=timeout, connect_timeout=connect_timeout, ssl_params=ssl_params)
         return 'cqlsh {options} -e "{command}" {host} {port}'.format(options=options, command=command, host=host, port=port)
 
-    def run_cqlsh(self, cmd, keyspace=None, port=None, timeout=60, verbose=True, split=False, target_db_node=None, connect_timeout=5):
+    def run_cqlsh(self, cmd, keyspace=None, port=None, timeout=120, verbose=True, split=False, target_db_node=None, connect_timeout=5):
```
Comment (juliayakovlev) [inline sdcm/cluster.py]: Do we need to change it for all tests/functions?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -2211,13 +2211,13 @@ def _gen_cqlsh_cmd(self, command, keyspace, timeout, host, port, connect_timeout
...
         """Runs CQL command using cqlsh utility"""
         cmd = self._gen_cqlsh_cmd(command=cmd, keyspace=keyspace, timeout=timeout,
                                   host=self.ip_address if not target_db_node else target_db_node.ip_address,
                                   port=port if port else self.CQL_PORT,
                                   connect_timeout=connect_timeout)
-        cqlsh_out = self.remoter.run(cmd, timeout=timeout + 5,  # we give 5 seconds to cqlsh timeout mechanism to work
+        cqlsh_out = self.remoter.run(cmd, timeout=timeout + 30,  # we give 5 seconds to cqlsh timeout mechanism to work
```
Comment (juliayakovlev) [inline sdcm/cluster.py]: Please, change the comment

File: sdcm/cluster.py (hunk 1 above)
Comment (roydahan) [reply sdcm/cluster.py]: I don't see a reason not to...

File: sdcm/cluster.py (hunk 2 above)
Comment (roydahan) [reply sdcm/cluster.py]: :(

**Pull request merged**